mlos_bench/mlos_bench/environments/remote/remote_env.py

```diff
@@ -175,10 +176,14 @@ def _remote_exec(self, script: Iterable[str]) -> Tuple[Status, datetime, Optiona
             Status is one of {PENDING, SUCCEEDED, FAILED, TIMED_OUT}
         """
         env_params = self._get_env_params()
+        command_name = re.sub(r"\W+", "-", self.name).lower()
```

Comment:
Also, precompile your regex:
```suggestion
    _RE_SPECIAL = re.compile(r"\W+")  # class-level const
        command_name = self._RE_SPECIAL.sub("-", self.name).lower()
```

Reply:
Updated to use precompiled regex